deployment/legacy-migration/maintenance-worker/migration/neo4j/import.sh

```diff
@@ -1,17 +1,48 @@
 #!/usr/bin/env bash
 set -e
 
-SECONDS=0
-SCRIPT_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-
-echo "MATCH (n) DETACH DELETE n;" | cypher-shell
+# import .env config
+set -o allexport
+source $(dirname "$0")/.env
+set +o allexport
 
-for collection in "badges" "categories" "users" "follows" "contributions" "shouts" "comments"
-do
-  for chunk in /tmp/mongo-export/splits/$collection/*
+# Import collection function defintion
+function import_collection () {
+  for chunk in ${IMPORT_PATH}splits/$1/*
   do
-    mv $chunk /tmp/mongo-export/splits/current-chunk.json
-    echo "Import ${chunk}" && cypher-shell < $SCRIPT_DIRECTORY/$collection.cql
+    mv $chunk ${IMPORT_CHUNK_PATH}
+    NEO4J_COMMAND="$(envsubst '${IMPORT_CHUNK_PATH_CQL}' < $(dirname "$0")/$1.cql)"
+    echo "Import ${chunk}"
+    echo "${NEO4J_COMMAND}" | "${IMPORT_CYPHERSHELL_BIN}" -u ${NEO4J_USERNAME} -p ${NEO4J_PASSWORD}
   done
-done
+}
+
+# Time variable
+SECONDS=0
+
+# Delete all Neo4J Database content
+echo "Deleting Database Contents"
+echo "MATCH (n) DETACH DELETE n;" | "${IMPORT_CYPHERSHELL_BIN}" -u ${NEO4J_USERNAME} -p ${NEO4J_PASSWORD}
+
+# Import Data
+echo "Start Importing Data"
+import_collection "badges"
+import_collection "categories"
+import_collection "users"
+import_collection "follows"
+import_collection "contributions"
+import_collection "shouts"
+import_collection "comments"
+#import_collection "emotions"
+#import_collection "invites"
+#import_collection "notifications"
+#import_collection "organizations"
+#import_collection "pages"
+#import_collection "projects"
+#import_collection "settings"
+#import_collection "status"
+#import_collection "systemnotifications"
+#import_collection "userscandos"
+#import_collection "usersettings"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-28T22:10:12Z" title="Wednesday, May 29th 2019, 12:10:12 am +02:00">May 29, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,17 +1,48 @@
 #!/usr/bin/env bash
 set -e
 
-SECONDS=0
-SCRIPT_DIRECTORY="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-
-echo "MATCH (n) DETACH DELETE n;" | cypher-shell
+# import .env config
+set -o allexport
+source $(dirname "$0")/.env
+set +o allexport
 
-for collection in "badges" "categories" "users" "follows" "contributions" "shouts" "comments"
-do
-  for chunk in /tmp/mongo-export/splits/$collection/*
+# Import collection function defintion
+function import_collection () {
+  for chunk in ${IMPORT_PATH}splits/$1/*
   do
-    mv $chunk /tmp/mongo-export/splits/current-chunk.json
-    echo "Import ${chunk}" && cypher-shell < $SCRIPT_DIRECTORY/$collection.cql
+    mv $chunk ${IMPORT_CHUNK_PATH}
+    NEO4J_COMMAND="$(envsubst '${IMPORT_CHUNK_PATH_CQL}' < $(dirname "$0")/$1.cql)"
+    echo "Import ${chunk}"
+    echo "${NEO4J_COMMAND}" | "${IMPORT_CYPHERSHELL_BIN}" -u ${NEO4J_USERNAME} -p ${NEO4J_PASSWORD}
   done
-done
+}
+
+# Time variable
+SECONDS=0
+
+# Delete all Neo4J Database content
+echo "Deleting Database Contents"
+echo "MATCH (n) DETACH DELETE n;" | "${IMPORT_CYPHERSHELL_BIN}" -u ${NEO4J_USERNAME} -p ${NEO4J_PASSWORD}
+
+# Import Data
+echo "Start Importing Data"
+import_collection "badges"
+#import_collection "categories"
+#import_collection "comments"
+#import_collection "contributions"
+#import_collection "emotions"
+#import_collection "follows"
+#import_collection "invites"
+#import_collection "notifications"
+#import_collection "organizations"
+#import_collection "pages"
+#import_collection "projects"
+#import_collection "settings"
+#import_collection "shouts"
+#import_collection "status"
+#import_collection "systemnotifications"
+#import_collection "users"
+#import_collection "userscandos"
+#import_collection "usersettings"
```


I cannot approve these changes. I would like to preserve the status quo, which is a lot more than just badges.